Quesnelia Updates

c-tobias opened the pull request:
Q release updates and revisions for Bulk edit, Consortium manager, Lists, and Settings > Consortium manager docs.

c-tobias committed:
Update _index.md

c-tobias committed:
Update _index.md

Lists - Quesnelia updates

c-tobias committed:
Update settings_consortium-manager.md

Settings > Consortium manager - Quesnelia updates

c-tobias committed:
Update _index.md

Consortium manager - Quesnelia updates

c-tobias committed:
Update _index.md

Bulk edit - Quesnelia updates

c-tobias committed:
Update _index.md

Q updates for Users.

c-tobias committed:
Q updates for Users, Settings-Users, ERM-Agreements, eHoldings, Licenses.

c-tobias committed:
Update _index.md

Corrections to Bulk edit (Local) section.

julianladisch commented on content/en/docs/ERM/Agreements/_index.md:
```diff
@@ -71,11 +72,11 @@ Keyboard shortcuts allow you to perform actions in this app using the keyboard.
+*   **Reason for closure.** This option only becomes available if **Closed** is selected from the **Status** drop-down list. Select from the drop-down list why the agreement was closed. You can configure the values that appear in the drop-down list in [Settings \> Agreements \> Pick list values](../settings/#settings_agreements/settings--agreements--pick-list-values).
+*   **Renewal priority.** Select from the drop-down list the level of interest in renewing the agreement. You can configure the values that appear in the drop-down list in [Settings \> Agreements \> Pick list values](../settings/#settings_agreements/settings--agreements--pick-list-values).
+*   **Is perpetual.** If the agreement provides perpetual access to the purchased materials, select *Yes*. You can configure the labels that appear in the drop-down list in [Settings \> Agreements \> Pick list values](../settings/#settings_agreements/settings--agreements--pick-list-values).
+*   **Content type.** If the agreement represents a particular content type or types, add one or more content type field and select a type from the drop-down list. You can configure the values that appear in the drop-down list in [Settings \> Agreements \> Pick list values](../settings/#settings_agreements/settings--agreements--pick-list-values).
```
The old links work properly: https://quesnelia.docs.folio.org/docs/erm/agreements/#primary-fields

The new links are broken links because of missing `../` and because of `#` at the wrong place.

c-tobias replied:
I've corrected the broken links for Settings > Agreements > Pick list values in ERM - Agreements.

c-tobias committed:
Update _index.md

Fixed broken links.

c-tobias committed:
Merge branch 'folio-org:quesnelia' into quesnelia

c-tobias committed:
Update _index.md

Corrected broken links for Settings > Agreements > Pick list values.

julianladisch commented on content/en/docs/ERM/Agreements/_index.md:
```diff
@@ -53,14 +52,16 @@ If you want to eUsage data in an agreement record, you also need to have the fol
 
 
 ##Keyboard shortcuts
```
Please add a space after `##` so that it becomes a section title.

c-tobias replied:
Done.

julianladisch commented on content/en/docs/ERM/Agreements/_index.md:
```diff
@@ -53,14 +52,16 @@ If you want to eUsage data in an agreement record, you also need to have the fol
-Keyboard shortcuts allow you to perform actions in this app using the keyboard.  See [Platform essentials > Keyboard shortcuts](< ref "/platform_essentials_keyboard_shortcuts.md" >) for more information.
+
+
+Keyboard shortcuts allow you to perform actions in this app using the keyboard.  See [Platform essentials \> Keyboard shortcuts](../../platform-essentials/keyboard-shortcuts) for more information.
```
The link `(../../platform-essentials/keyboard-shortcuts)` doesn't work. Please pick one of these:

* `(< ref "keyboardshortcuts.md" >)`
* `(../../platform-essentials/keyboard-shortcuts/keyboardshortcuts)`

c-tobias replied:
Corrected with (< ref "keyboardshortcuts.md" >)

julianladisch commented on content/en/docs/ERM/Agreements/_index.md:
```diff
@@ -86,7 +87,7 @@ Note: Adding a content type is optional, but if you click **Add content type**,
 
 1. Click **Add content type**.
-2. Select the agreement record’s content type from the drop-down list. You can configure the values that appear in the drop-down list in [Settings > Agreements > Pick list values]({{< ref "/settings_agreements.md#settings--agreements--pick-list-values" >}}).
+2. Select the agreement record’s content type from the drop-down list. You can configure the values that appear in the drop-down list in [Settings \> Agreements \> Pick list values](/../../settings/settings_agreements/settings_agreements/#settings--agreements--pick-list-values).
```
The link `(/../../settings/settings_agreements/settings_agreements/#settings--agreements--pick-list-values)` doesn't work.

Please use one of these:
* `{{< ref "settings_agreements#settings--agreements--pick-list-values" >}}`
* `{{< ref "settings_agreements.md#settings--agreements--pick-list-values" >}}`
* `{{< ref "/settings_agreements.md#settings--agreements--pick-list-values" >}}`
* `(../../settings/settings_agreements/settings_agreements/#settings--agreements--pick-list-values)` (without preceding `/`)

c-tobias replied:
Corrected with {{< ref "settings_agreements.md#settings--agreements--pick-list-values" >}}.

julianladisch replied:
Sorry, I forgot the round brackets/parentheses, it should be
* `({{< ref "/settings_agreements.md#settings--agreements--pick-list-values" >}})`

c-tobias replied:
Corrected the link with (../../settings/settings_agreements/settings_agreements/#settings--agreements--pick-list-values) instead of the ({{< ref>}}) format.

julianladisch commented on content/en/docs/ERM/Agreements/_index.md:
```diff
@@ -460,7 +461,7 @@ To filter agreements by their reason for closure, select one of the listed optio
 *   **Superseded.** Agreements that have been superseded by another agreement.
 
-Renewal priority values are configured in [Settings > Agreements > Pick list values]({{< ref "/settings_agreements.md#settings--agreements--pick-list-values" >}}).
+Renewal priority values are configured in [Settings > Agreements > Pick list values](../../settings_agreements/settings_agreements/#settings--agreements--pick-list-values").
```
The old link `({{< ref "/settings_agreements.md#settings--agreements--pick-list-values" >}})` works: https://docs.folio.org/docs/erm/agreements/#reason-for-closure

The new link `(../../settings_agreements/settings_agreements/#settings--agreements--pick-list-values")` doesn't work.

Please use one of these:
* `{{< ref "settings_agreements#settings--agreements--pick-list-values" >}}`
* `{{< ref "settings_agreements.md#settings--agreements--pick-list-values" >}}`
* `{{< ref "/settings_agreements.md#settings--agreements--pick-list-values" >}}`
* `(../../settings/settings_agreements/settings_agreements/#settings--agreements--pick-list-values)`

c-tobias replied:
Corrected with {{< ref "/settings_agreements.md#settings--agreements--pick-list-values" >}}.

julianladisch replied:
Sorry, I forgot the round brackets/parentheses, it should be
* `({{< ref "/settings_agreements.md#settings--agreements--pick-list-values" >}})`

c-tobias replied:
Corrected the links using the (../../settings/settings_agreements/settings_agreements/#settings--agreements--pick-list-values) format instead of the ({< ref >}) format.

julianladisch commented on content/en/docs/ERM/Agreements/_index.md:
```diff
@@ -472,7 +473,7 @@ To filter agreements by their renewal priority, select one of the listed options
 *   **Definitely cancel.** Agreements you do not want to renew.
 
-Renewal priority values are configured in [Settings > Agreements > Pick list values]({{< ref "/settings_agreements.md#settings--agreements--pick-list-values" >}}).
+Renewal priority values are configured in [Settings \> Agreements \> Pick list values](../../settings_agreements/settings_agreements/#settings--agreements--pick-list-values).
```
The old link `({{< ref "/settings_agreements.md#settings--agreements--pick-list-values" >}})` works: https://docs.folio.org/docs/erm/agreements/#renewal-priority

The new link `(../../settings_agreements/settings_agreements/#settings--agreements--pick-list-values)` doesn't work.

Please use one of these:
* `{{< ref "settings_agreements#settings--agreements--pick-list-values" >}}`
* `{{< ref "settings_agreements.md#settings--agreements--pick-list-values" >}}`
* `{{< ref "/settings_agreements.md#settings--agreements--pick-list-values" >}}`
* `(../../settings/settings_agreements/settings_agreements/#settings--agreements--pick-list-values)`

c-tobias replied:
Corrected with {{< ref "/settings_agreements.md#settings--agreements--pick-list-values" >}}.

julianladisch replied:
Sorry, I forgot the round brackets/parentheses, it should be
* `({{< ref "/settings_agreements.md#settings--agreements--pick-list-values" >}})`

c-tobias replied:
Corrected the link with (../../settings/settings_agreements/settings_agreements/#settings--agreements--pick-list-values). I could not get the ({{< ref >}}) format to work.

c-tobias committed:
Update _index.md

Corrected links in ERM > Agreements.

julianladisch commented on content/en/docs/ERM/Agreements/_index.md:
```diff
@@ -86,7 +87,7 @@ Note: Adding a content type is optional, but if you click **Add content type**,
 
 1. Click **Add content type**.
-2. Select the agreement record’s content type from the drop-down list. You can configure the values that appear in the drop-down list in [Settings > Agreements > Pick list values]({{< ref "/settings_agreements.md#settings--agreements--pick-list-values" >}}).
+2. Select the agreement record’s content type from the drop-down list. You can configure the values that appear in the drop-down list in [Settings \> Agreements \> Pick list values]{{< ref "settings_agreements.md#settings--agreements--pick-list-values" >}}.
```
(Sorry, my advice five days ago was misleading.)

julianladisch commented on content/en/docs/ERM/Agreements/_index.md:
```diff
@@ -1213,5 +1245,5 @@ You can add and assign notes to agreement records, agreement lines, and e-resour
 The Dashboard app is designed to enable a personalized view of key information from across FOLIO apps at a glance. In its first release (included in the Juniper flower release), the Dashboard includes the ability to display information from the Agreements and Licenses applications.
 
-In the Morning Glory release, the Agreements app comes with two dashboard widgets ("ERM Agreements" and "ERM Agreement Jobs"). There are examples of how you can use these two 
+In the Quesnelia release, the Agreements app comes with two dashboard widgets ("ERM Agreements" and "ERM Agreement Jobs"). There are examples of how you can use these two 
```
Can "In the Quesnelia release, " been deleted to avoid updating the flower release name?

c-tobias replied:
Deleted.

julianladisch commented on content/en/docs/ERM/Licenses/_index.md:
```diff
@@ -135,10 +135,10 @@ Organizations must have an organization record created in the Organizations app
 4. (Optional) Filter results by Organizations status, Tags, Is vendor, Country, Languages, or Payment method.
 5. In the Organizations pane, click the **organization** to select it. The Select Organization dialog closes and the organization is added to the license record.
-6. Select a **Role** for the organization from the drop-down list. Roles are configured in the Settings app under [Licenses > License term pick list setup]({{< ref "/settings_licenses.md#settings--licenses--pick-lists" >}}).
+6. Select a **Role** for the organization from the drop-down list. Roles are configured in the Settings app under [Licenses > Pick list setup](../../settings/settings_licenses/settings_licenses/settings--licenses--pick-lists).
```
The link
`[Licenses > Pick list setup](../../settings/settings_licenses/settings_licenses/settings--licenses--pick-lists)`
doesn't work because the `#` is missing. Use this instead:
`[Licenses > Pick list setup](../../settings/settings_licenses/settings_licenses/#settings--licenses--pick-lists)`

c-tobias replied:
Corrected as suggested.

julianladisch commented on content/en/docs/ERM/Licenses/_index.md:
```diff
@@ -191,7 +191,7 @@ If a core document is present in a license record, then a **copy icon** will app
 ### Terms
 
-Terms are the terms of use that define what you can or cannot do with the licensed content. In FOLIO, you can define the Terms that appear in license records through configurations in the Settings app. The Settings app is where you establish the term label, the type of term, default visibility of the term, and term status (primary or optional). For more information, see [Settings > Licenses > Ter ...
+Terms are the terms of use that define what you can or cannot do with the licensed content. In FOLIO, you can define the Terms that appear in license records through configurations in the Settings app. The Settings app is where you establish the term label, the type of term, default visibility of the term, and term status (primary or optional). For more information, see [Settings \> Licenses \> T ...
```
This link doesn't work because the `#` is missing.

c-tobias replied:
# added to link.

julianladisch commented on content/en/docs/ERM/Licenses/_index.md:
```diff
@@ -240,7 +240,7 @@ A supplementary document is any additional document relevant to the license reco
 1. Click **Add supplementary document.**
 2. Enter a **Name** in the box.
-3. Select a **Category** from the drop-down list. Categories for Supplementary documents can be configured in the Settings app under [Licenses > License term pick list setup]({{< ref "/settings_licenses.md#settings--licenses--pick-lists" >}}).
+3. Select a **Category** from the drop-down list. Categories for Supplementary documents can be configured in the Settings app under [Licenses > License term pick list setup](../../settings_licenses/settings_licenses/settings--licenses--pick-lists).
```
This link doesn't work because `settings/` and the `#` are missing.
This link works:
* `[Licenses > License term pick list setup](../../settings/settings_licenses/settings_licenses/#settings--licenses--pick-lists)`

c-tobias replied:
Corrected link as suggested.

julianladisch commented on content/en/docs/ERM/Licenses/_index.md:
```diff
@@ -275,7 +275,7 @@ To filter licenses by their status, in the **Search & filter** pane, click **Sta
 *   **Expired.** Licenses no longer in use by your library.
 
-Status values are configured in [Settings > Licenses > Pick list values]({{< ref "/settings_licenses.md#settings--licenses--pick-list-values" >}}).
+Status values are configured in [Settings > Licenses > Pick list values](../../settings/settings_licenses/settings_licenses/settings--licenses/settings--licenses--pick-list-values).
```
This link doesn't work.
Use this instead:
* `[Settings \> Licenses \> Pick list values](../../settings/settings_licenses/settings_licenses/#settings--licenses--pick-list-values)`

c-tobias replied:
Corrected link as suggested.

julianladisch commented on content/en/docs/ERM/Licenses/_index.md:
```diff
@@ -311,7 +311,7 @@ To filter licenses by their associated organizations’ roles, follow these step
 4. Select the role you want to filter by. The search results appear in the Licenses pane.
 
-Organization role values are configured in [Settings > Licenses > Pick list values]({{< ref "/settings_licenses.md#settings--licenses--pick-list-values" >}}).
+Organization role values are configured in [Settings > Licenses > Pick list values](../../settings/settings_licenses/settings_licenses/settings--licenses/settings--licenses--pick-list-values).
```
This link doesn't work.
Use this instead:
* `[Settings \> Licenses \> Pick list values](../../settings/settings_licenses/settings_licenses/#settings--licenses--pick-list-values)`

c-tobias replied:
Corrected link as suggested.

julianladisch commented on content/en/docs/ERM/Licenses/_index.md:
```diff
@@ -355,6 +355,53 @@ To filter licenses by their terms, follow these steps:
+7. (Optional) To search for licenses using multiple supplementary document filters, click **Add filter.** Note: To delete a rule or filter in the Supplementary document filter builder, click on the **trash can icon** next to the rule or filter.
+8. Click **Save & close.** The search results appear in the Licenses pane.
+
+Document values are configured in [Settings > Licenses > Pick list values](../../settings/settings_licenses_settings_licenses/settings--license--pick-list-values).
```
This link doesn't work.
Use this instead:
* `[Settings \> Licenses \> Pick list values](../../settings/settings_licenses/settings_licenses/#settings--licenses--pick-list-values)`

c-tobias replied:
Corrected link as suggested.

julianladisch commented on content/en/docs/ERM/Licenses/_index.md:
```diff
@@ -355,6 +355,53 @@ To filter licenses by their terms, follow these steps:
+7. (Optional) To search for licenses using multiple core document filters, click **Add filter.** Note: To delete a rule or filter in the Core document filter builder, click on the **trash can icon** next to the rule or filter.
+8. Click **Save & close.** The search results appear in the Licenses pane.
+
+Document values are configured in [Settings > Licenses > Pick list values](../../settings/settings_licenses_settings_licenses/settings--license--pick-list-values).
```
This link doesn't work.
Use this instead:
* `[Settings \> Licenses \> Pick list values](../../settings/settings_licenses/settings_licenses/#settings--licenses--pick-list-values)`

c-tobias replied:
Corrected link as suggested.

julianladisch commented on content/en/docs/ERM/Licenses/_index.md:
```diff
@@ -527,4 +585,4 @@ You can add and assign notes to license records. Assigning a note means you are
 The Dashboard app is designed to enable a personalized view of key information from across FOLIO apps at a glance. In its first release (included in the Juniper flower release), the Dashboard includes the ability to display information from the Agreements and Licenses applications.
 
-In the Morning Glory release, the Licenses app comes with one dashboard widget ("ERM Licenses"). There are examples of how you can use this  widget definition to achieve a wide range of outcomes in the Example widget configurations documentation.
+In the Quesnelia release, the Licenses app comes with one dashboard widget ("ERM Licenses"). There are examples of how you can use this  widget definition to achieve a wide range of outcomes in the Example widget configurations documentation.
```
Can "In the Quesnelia release, " been deleted to avoid updating the name whenever a new flower release gets published?

c-tobias replied:
Deleted.

c-tobias replied:
Deleted reference to Q release.

julianladisch commented on content/en/docs/Settings/Settings_consortium-manager/settings_consortium-manager.md:
```diff
@@ -34,6 +34,7 @@ The following are the permissions for **Settings \> Consortium Manager**:
 * **Settings (Consortia): Can view and edit consortia membership** This permission allows the user to view and edit the details of the list of members in the Settings app of the central tenant.
 * **Settings (Consortia): Can view consortia membership** This permission allows the user to view the details of the list of members in the Settings app of the central tenant.
 
+For more general information about permissions, see [Platform Essentials > Permissions](../platform-essentials/permissions/).
```
This link doesn't work, use this instead:
* `[Platform Essentials > Permissions](../../../platform-essentials/permissions/)`

c-tobias replied:
Corrected link.

c-tobias committed:
Update _index.md

Link corrections as suggested by Julian.

c-tobias committed:
Update _index.md

Corrections suggested by Julian. Note that I used (../../settings/settings_agreements/settings_agreements/#settings--agreements--pick-list-values) instead of the ({< ref >}) format to correct the links.

c-tobias committed:
Update _index.md

Corrected links.

c-tobias committed:
Update _index.md

Deleted "Quesnelia release".

c-tobias committed:
Update settings_consortium-manager.md

Corrected link.
